libraries/botbuilder-ai/src/luisAdaptiveRecognizer.ts

```diff
@@ -147,17 +174,43 @@ export class LuisAdaptiveRecognizer extends Recognizer implements LuisAdaptiveRe
      * @returns {LuisRecognizerOptionsV3} luis recognizer options
      */
     public recognizerOptions(dialogContext: DialogContext): LuisRecognizerOptionsV3 {
-        const options = Object.assign({}, this.predictionOptions);
+        const options: LuisRecognizerOptionsV3 = {
+            apiVersion: 'v3',
+            externalEntityRecognizer: this.externalEntityRecognizer,
+            telemetryClient: this.telemetryClient,
+            includeAllIntents: false,
+            includeInstanceData: false,
+            includeAPIResults: false,
+            log: true,
```

Comment:
Should this be based on the log personal information flag?

Reply:
I was referring default values from https://github.com/microsoft/botbuilder-dotnet/blob/main/libraries/Microsoft.Bot.Builder.AI.LUIS/V3/LuisPredictionOptions.cs because we cannot define default values for interfaces:
C#:
```
AI.LuisV3.LuisPredictionOptions options = new LuisV3.LuisPredictionOptions();
```
JS (ported):
```
const options: LuisRecognizerOptionsV3 = {
    apiVersion: 'v3',
    externalEntityRecognizer: this.externalEntityRecognizer,
    telemetryClient: this.telemetryClient,
    includeAllIntents: false,
    includeInstanceData: false,
    includeAPIResults: false,
    log: true,
    preferExternalEntities: true,
    slot: 'production',
};
```